compiler/src/main/kotlin/edu/cornell/cs/apl/viaduct/codegeneration/BackendCodeGenerator.kt

```diff
@@ -338,3 +263,89 @@ class BackendCodeGenerator(
             CodeBlock.of("%N.%M(%L, %N)", "runtime", sendMember, value, receiver.name)
```

Comment:
Why are `send` and `receive` in the `Context` class? It seems misplaced. I thought the context provides names and that's it.

Reply:
It’s so that code generators don’t have to hardcode the variable name “runtime”. So they are here for naming purposes.

Reply:
I think it would make more sense for the `Context` object to expose the name "runtime", like `Context.runtime` or something like that. This is a style thing though

Reply:
That could also work. My idea was to reduce code duplication. The send/receive code looks the same everywhere, so it felt like bad practice to have to reconstruct `Context.runtime` applied to `"send"` with the correct arguments (type and value) in the correct places everywhere. This also let's us easily change the way we call these functions. E.g., if we made the type argument of `receive` a value argument, no code needs to change.

This was introduces in another PR though, so we should discuss it separately.